widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.java

```diff
@@ -216,31 +216,31 @@ public void startChat(
             boolean useOverlays,
             Bundle savedInstanceState) {
         Activity activity = Utils.getActivity(this.getContext());
-        boolean isConfigurationChange = activity instanceof ChatActivity &&
-                savedInstanceState != null;
-        if (!isConfigurationChange) {
-            if (chatHeadsController != null) {
-                chatHeadsController.onNavigatedToChat(
-                        new ChatHeadInput(
-                                companyName,
-                                queueId,
-                                contextUrl,
-                                this.theme
-                        ),
-                        activity instanceof ChatActivity,
-                        activity instanceof ChatActivity && useOverlays
-                );
-            }
-        }
         if (controller != null) {
-            controller.initChat(
-                    companyName,
-                    queueId,
-                    contextUrl,
+
+            controller.initChat(companyName, queueId, contextUrl);
+
+            controller.updatePermissions(
                     Settings.canDrawOverlays(this.getContext()),
                     NotificationManager.areNotificationsEnabled(this.getContext(), NotificationFactory.NOTIFICATION_CALL_CHANNEL_ID),
                     NotificationManager.areNotificationsEnabled(this.getContext(), NotificationFactory.NOTIFICATION_SCREEN_SHARING_CHANNEL_ID)
             );
+
+            if (activity instanceof ChatActivity && savedInstanceState == null) {
+                if (chatHeadsController != null) {
+                    chatHeadsController.onNavigatedToChat(
+                            new ChatHeadInput(
+                                    companyName,
+                                    queueId,
+                                    contextUrl,
+                                    this.theme
+                            ),
+                            activity instanceof ChatActivity,
+                            activity instanceof ChatActivity && useOverlays,
+                            controller.isMediaQueueingOnGoing()
```

Comment:
Is needed so the chatHeadsController knows of an ongoing media queueing. Start audio call / Start video call queueing.